src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -15,9 +19,13 @@
     }
 
     private User findByLogin(String login) {
+        logger.debug("Method findByLogin was called with login={}", login);
         User user = new User(login, "1234");
         // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
+        logger.debug("New user was created with login: {}, with ID: {}. Data sent to DB",
```

Comment:
The log message should be more informative and should not suggest that data was sent to the database if it was not. Instead, you could log that a new user object was created. Example: logger.debug("New user object created with login: {}, ID: {}", login, user.getUserId());